openapi3.yaml

```diff
@@ -61,12 +147,37 @@ components:
       properties:
         message:
           type: string
-    resource:
+    requestBodyForFilter:
+      type: object
+      required:
+        - arrayOfilterProduct
+      properties:
+        arrayOfilterProduct:
+          type: array
+          items:
+            $ref: '#/components/schemas/filterProduct'
+        limit:
+          type: integer
+          minimum: 0
+        skip:
+          type: integer
+          minimum: 0
+    filterProduct:
+              # expmple: {feild:'minZoom',oprator:'>',filterArg:'10'}
       type: object
       required:
-        - id
-        - name
-        - description
+        - feild
+        - oprator
+        - filterArg
+      properties:
+        feild:
+          type: string
+        operator:
+          type: string
+        filterArg:
```

Comment:
Do you mean `filterValue`? Isn't `value` better?

Reply:
agree, changed that.